server/runtime/src/main/server/bin/report.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+
+PROGNAME=$(basename "$0")
+DIRNAME=$(dirname "$0")
+
+count() { echo $#; }
+
+# Get the PID
+if [ "x$1" = "x" ]; then
+  # There may be more than one
+  PPID=$(pgrep server.sh)
+  PPID_COUNT=$(count $PPID)
+  if [ "$PPID_COUNT" -gt "1" ]; then
+    printf "Multiple processes detected. Specify one of the following PIDs as the first argument:\n %s" "$PPID"
+    exit 1
+  fi
+  PID=$(pgrep -P $PPID)
+else
+  PID=$1
+  shift
+fi
+
+TMPDIR=$(mktemp --tmpdir -d infinispan-server.XXXXXX)
+
+# Gather information about the system
+cp /etc/os-release "$TMPDIR/os-release"
+ip address > "$TMPDIR/ip-address"
+ip route > "$TMPDIR/ip-route"
+ip maddress > "$TMPDIR/ip-maddress"
+ip mroute > "$TMPDIR/ip-maddress"
+cat /proc/cpuinfo > "$TMPDIR/cpuinfo"
+cat /proc/meminfo > "$TMPDIR/meminfo"
+uname -a > "$TMPDIR/uname"
+df -h > "$TMPDIR/df"
+ss -t -a > "$TMPDIR/ss-tcp"
+ss -u -a > "$TMPDIR/ss-udp"
+lsof / > "$TMPDIR/lsof" 2> /dev/null
+
+# Setup ISPN_HOME
+ISPN_HOME=$(cd "$DIRNAME/.." > /dev/null; pwd)
+
+# Setup ISPN_ROOT
+if [ "x$1" = "x" ]; then
```

Comment:
old code?

Reply:
ignore me...

Reply:
No, it's needed to explicitly pass in a different root